src/components/common/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -147,7 +147,8 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
   /** Watch for form submission and emit results. */
   @Watch('formType')
   private async onSubmit (): Promise<any> {
-    if (this.formType === CorrectionTypes.CORRECT_NEW_NR) {
+    // safety check
+    if (this.formType === NameChangeOptions.CORRECT_NEW_NR) {
```

Comment:
This is a safety check because this component should only see this form type.

Reply:
A comment to explain the comment?

Reply:
I was thinking on line 150  just expanding on the safety check with what you have written here. It might be overkill.

Reply:
I'll do it.

Reply:
Done.